franka_joint_torque_controller:
    type: effort_controllers/JointGroupEffortController
    arm_id: panda
    joints:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
    publish_rate: 10.0
    coriolis_factor: 1.0